packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -0,0 +1,713 @@
+  double get _lastVisibleLayoutOffset {
+    assert(hasSize);
+    if (_renderChildrenOutsideViewport)
+      return double.infinity;
+    return _minScrollExtent + size.height / 2.0 + _itemExtent / 2.0 + offset.pixels;
+  }
+
+  /// Given the _diameterRatio, return the largest absolute angle of the item
+  /// at the edge of the portion of the visible cylinder.
+  ///
+  /// For a _diameterRatio of 1 or less than 1 (i.e. the viewport is bigger
+  /// than the cylinder diameter), this value reaches and clips at pi / 2.
+  ///
+  /// When the center of children passes this angle, they are no longer painted
+  /// if renderChildrenOutsideViewport is false.
+  double get _maxVisibleRadian {
+    if (_diameterRatio < 1.0)
+      return math.pi / 2.0;
+    return math.asin(1 / _diameterRatio);
+  }
+
+  double _getIntrinsicCrossAxis(_ChildSizingFunction childSize) {
+    double extent = 0.0;
+    RenderBox child = firstChild;
+    while (child != null) {
+      extent = math.max(extent, childSize(child));
+      final _ListWheelParentData childParentData = child.parentData;
+      child = childParentData.nextSibling;
+    }
+    return extent;
+  }
+
+  @override
+  double computeMinIntrinsicWidth(double height) {
+    return _getIntrinsicCrossAxis(
+      (RenderBox child) => child.getMinIntrinsicWidth(height)
+    );
+  }
+
+  @override
+  double computeMaxIntrinsicWidth(double height) {
+    return _getIntrinsicCrossAxis(
+      (RenderBox child) => child.getMaxIntrinsicWidth(height)
+    );
+  }
+
+  @override
+  double computeMinIntrinsicHeight(double width) {
+    if (childCount > 0)
+      return childCount * _itemExtent;
+    return 0.0;
+  }
+
+  @override
+  double computeMaxIntrinsicHeight(double width) {
+    if (childCount > 0)
+      return childCount * _itemExtent;
+    return 0.0;
+  }
+
+  @override
+  bool get sizedByParent => true;
+
+  @override
+  void performResize() {
+    size = constraints.biggest;
+  }
+
+  // We don't override computeDistanceToActualBaseline(), because we
+  // want the default behavior (returning null). Otherwise, as you
+  // scroll, it would shift in its parent if the parent was baseline-aligned,
+  // which makes no sense.
+
+  @override
+  void performLayout() {
+    double currentOffset = 0.0;
+    RenderBox child = firstChild;
+    final BoxConstraints innerConstraints =
+        constraints.copyWith(
+          minHeight: _itemExtent,
+          maxHeight: _itemExtent,
+          minWidth: 0.0,
+        );
+    while(child != null) {
+      child.layout(innerConstraints, parentUsesSize: true);
+      final _ListWheelParentData childParentData = child.parentData;
+      // Centers the child in the cross axis. Consider making it configurable.
+      final double crossPosition = size.width / 2.0 - child.size.width / 2.0;
+      childParentData.offset = new Offset(crossPosition, currentOffset);
+      currentOffset += _itemExtent;
+      child = childParentData.nextSibling;
```

Comment:
i think you can just do `child = nextChild(child)`

Reply:
Done. Though it seems like it does the same thing